### PR TITLE
feat(ramses-cl): Adjust for Ramses CL

### PR DESCRIPTION
**Pull request by hochbruj**

### CODE REVIEW

File: src/test/ramsesCL.test.ts [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        await pool.claimFees(Dapp.RAMSESCL, tokenId);
+        expect((await balanceDelta(pool.address, USDC, pool.signer)).gt(0));
+      });
+
+      it("get rewards of a CL position", async () => {
+        await provider.send("evm_increaseTime", [24 * 3600 * 3]); // 1 day
+        await provider.send("evm_mine", []);
+        await pool.getRewards(Dapp.RAMSESCL, tokenId, [RAM]);
+        expect((await balanceDelta(pool.address, RAM, pool.signer)).gt(0));
+      });
```
Comment (kent426) [inline src/test/ramsesCL.test.ts:139]: we probably can add a 100% decreaseLiquidity test at the end.

**Review by kent426 — CHANGES_REQUESTED** [review]
need to add `getRewards` in  `getDecreaseLiquidityTxData` for `Dapp.RAMSESCL`

File: src/services/ramses/vesting.ts [hunk 2]
```diff
@@ -11,3 +15,10 @@ export function getCreateVestTxData(amount: BigNumber | string): string {
 export function getExitVestTxData(vestId: number): string {
   return iXRam.encodeFunctionData("exitVest", [vestId, false]);
 }
+
+export function getRewardsTxDta(tokrnId: string, rewards: string[]): string {
+  return iRamsesNonfungiblePositionManager.encodeFunctionData("getReward", [
+    tokrnId,
+    rewards
+  ]);
+}
```
[the remark this answers is not in the document]
Comment (hochbruj) [reply src/services/ramses/vesting.ts]: fixed [f9f078e](https://github.com/dhedge/dhedge-v2-sdk/pull/63/commits/f9f078ed20ccc58cf1aaebc0c7b5db5d08ace2be)

File: src/services/uniswap/V3Liquidity.ts [hunk 3]
```diff
@@ -221,7 +230,8 @@ export async function getDecreaseLiquidityTxData(
   if (
     dapp === Dapp.UNISWAPV3 ||
     dapp === Dapp.VELODROMECL ||
-    dapp === Dapp.AERODROMECL
+    dapp === Dapp.AERODROMECL ||
+    dapp === Dapp.RAMSESCL
```
[the remark this answers is not in the document]
Comment (hochbruj) [reply src/services/uniswap/V3Liquidity.ts:234]: added instruction in README [f9f078e](https://github.com/dhedge/dhedge-v2-sdk/pull/63/commits/f9f078ed20ccc58cf1aaebc0c7b5db5d08ace2be)

File: src/test/ramsesCL.test.ts (hunk 1 above)
Comment (hochbruj) [reply src/test/ramsesCL.test.ts:139]: added test [f9f078e](https://github.com/dhedge/dhedge-v2-sdk/pull/63/commits/f9f078ed20ccc58cf1aaebc0c7b5db5d08ace2be)

**Review by kent426 — APPROVED** [review]

**Pull request merged**